### PR TITLE
App/Language: Add a LLM example about text generation

### PR DESCRIPTION
**Pull request by niley7464**
This patch adds a draft of the LLM example.
It uses llama2 model to generate text using input prompt.

### CODE REVIEW

File: ml_inference_offloading/src/main/java/ai/nnstreamer/ml/inference/offloading/MainService.kt [hunk 1]
```diff
@@ -407,4 +417,39 @@ class MainService : Service() {
...
+        CoroutineScope(Dispatchers.IO).launch {
+            val service = findService("llama")
+
+            service?.let {
+                startService(it.serviceId)
+                // todo: Support other models
+                runLlama2(input, getIpAddress(isRunningOnEmulator), it.port, NewDataCb(messenger))
+            }?.run {
+                Log.e(TAG, "Not supported LLM")
+            }
```
Comment (wooksong) [inline ml_inference_offloading/src/main/java/ai/nnstreamer/ml/inference/offloading/MainService.kt:452]: When is this block invoked?

File: ml_inference_offloading/src/main/java/ai/nnstreamer/ml/inference/offloading/domain/LlamaUtil.kt [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+import org.nnsuite.nnstreamer.NNStreamer
+import org.nnsuite.nnstreamer.Pipeline
+import org.nnsuite.nnstreamer.TensorsData
+import org.nnsuite.nnstreamer.TensorsInfo
+import java.nio.ByteBuffer
+
+fun runLlama2(input: String, hostAddress: String, servicePort: Int, newDataCb: NewDataCb) {
+    val port = findPort()
+    val desc =
+        "appsrc name=srcx ! application/octet-stream ! tensor_converter ! other/tensors,format=flexible ! tensor_query_client host=${hostAddress} port=${port} dest-host=${hostAddress} dest-port=${servicePort} timeout=1000000 ! tensor_sink name=sinkx"
```
Comment (wooksong) [inline ml_inference_offloading/src/main/java/ai/nnstreamer/ml/inference/offloading/domain/LlamaUtil.kt:13]:
```suggestion
        "appsrc name=srcx ! application/octet-stream ! tensor_converter ! other/tensors,format=flexible ! tensor_query_client host=$hostAddress port=$port dest-host=$hostAddress dest-port=$servicePort timeout=1000000 ! tensor_sink name=sinkx"
```

File: ml_inference_offloading/src/main/java/ai/nnstreamer/ml/inference/offloading/MainService.kt (hunk 1 above)
Comment (niley7464) [reply ml_inference_offloading/src/main/java/ai/nnstreamer/ml/inference/offloading/MainService.kt:452]: I just want to check if there are no LLM related service.

File: ml_inference_offloading/src/main/java/ai/nnstreamer/ml/inference/offloading/domain/LlamaUtil.kt (hunk 2 above)
Comment (niley7464) [reply ml_inference_offloading/src/main/java/ai/nnstreamer/ml/inference/offloading/domain/LlamaUtil.kt]: Updated
Thank you 😅

File: ml_inference_offloading/src/main/java/ai/nnstreamer/ml/inference/offloading/MainService.kt (hunk 1 above)
Comment (niley7464) [reply ml_inference_offloading/src/main/java/ai/nnstreamer/ml/inference/offloading/MainService.kt:452]: https://tourspace.tistory.com/208

I was making mistake when using 'let' incorrectly.
```
1. When not to use LET
1-1. 불변 변수의 null check (When just checking null for an immutable variable, don't use LET)
```

**Pull request merged**